### PR TITLE
chore: switch to Web Apps WG

### PR DESCRIPTION
**Pull request by marcoscaceres**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/html-aria/pull/131.html" title="Last updated on May 16, 2019, 2:06 AM UTC (bafa916)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/html-aria/131/43d8df8...bafa916.html" title="Last updated on May 16, 2019, 2:06 AM UTC (bafa916)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,24 +4,23 @@
     <meta charset='utf-8'>
     <title>ARIA in HTML</title>
     <script src="https://www.w3.org/scripts/jquery/2.1/jquery.min.js"></script>
-    <script src="https://www.w3.org/Tools/respec/respec-w3c-common" class="remove"></script>
+    <script src="https://www.w3.org/Tools/respec/respec-w3c" class="remove"></script>
```
Comment (marcoscaceres) [inline index.html]: Faster 💨

File: index.html [hunk 2]
```diff
@@ -4,24 +4,23 @@
...
                     companyURL: "https://www.paciellogroup.com",
                     w3cid:      "35007" }
                     ],
-          wg:           "Web Platform Working Group",
-          wgURI:        "https://www.w3.org/WebPlatform/WG/",
-          wgPublicList: "public-html",
-          edDraftURI:   "https://w3c.github.io/html-aria/",
+          wg:           "Web Applications Working Group",
+          wgURI:        "https://www.w3.org/2019/webapps/",
+          wgPublicList: "public-webapps",
```
Comment (marcoscaceres) [inline index.html:22]: If you want only GitHub comments, get rid of this line.

**Comment by marcoscaceres** [comment]
Merging this, as it gets the spec into the right working group... @stevefaulkner, don't forget to join the new working group. You put the "s" in WebApps, after all.

**Review by Tabi666 — COMMENTED** [review]
#130

**Pull request merged**